README.md

```diff
@@ -1 +1,102 @@
-Test
\ No newline at end of file
+# Trustly Parliament Challenge
+
+### Problems
+1. Not being able to merger speeches data with members data
```

Comment:
Whatever problem you face, always try to solve the problem on a small scale. So basically what i mean by this is, take two dicts and try to merge them. Once you know how to do it, try to apply the same logic and solution on a big scale for the real problem.